src/store/channels/saga.ts

```diff
@@ -62,9 +90,19 @@ export function* clearChannels() {
   yield put(removeAll({ schema: schema.key }));
 }
 
+function* listenForChannelLoadedEvent() {
+  const channel = yield call(conversationsChannel);
+  while (true) {
+    const { channelId = undefined } = yield take(channel, ChannelEvents.MessagesLoadedForChannel);
+    if (channelId) {
+      yield call(markAllAsRead);
```

Comment:
I think maybe we should have 2 events. One for Channels and one for Conversations. I know they got merged together but we should treat them as totally different things.

Also, we shouldn't mark everything read like this. We should only mark the conversations as read IF this event was for the already open channel. The bug here is, what if we decided to start preloading messages for channels that we're not viewing at the moment? We'd get a bunch of these events but we wouldn't want to just "mark the current conversation" unread.